### ISSUE TITLE
Use Unicode ellipsis instead of three dots

### ISSUE THREAD

**Issue by M1dgard** [jaagr/polybar#1195]
In monospaced fonts, the three dots `...` are too wide: ![screenshot_2018-04-26_01-28-48](https://user-images.githubusercontent.com/4283376/39278025-72f9bce6-48e0-11e8-9722-66de7b1e6290.png)

The single glyph `…` looks better: ![screenshot_2018-04-26_01-32-52](https://user-images.githubusercontent.com/4283376/39278138-0683d082-48e1-11e8-92db-f15ec5f5cef9.png)

**Comment by patrick96** [comment]
We could maybe even add an option to the label to set the ellipsis string. But you're right, the the character ellipsis is quite long

**Comment by M1dgard** [comment]
Yes, being able to do
```ini
label-song-ellipsis = …
label-song-ellipsis = ...
label-song-ellipsis = ..
label-song-ellipsis = $
label-song-ellipsis = $
```
would be nice

**Comment by patrick96** [comment]
Exactly, but we can't name it `label-song-ellipsis`. That is already taken for the boolean.

> I don't know if it's currently taken into account, but make sure to subtract the length of the chosen ellipsis string from maxlen!

I think we already have logic for that in the token handling, so we can use that ;)

**Comment by M1dgard** [comment]
> Exactly, but we can't name it `label-song-ellipsis`. That is already taken for the boolean.

I think that #1194 proves that no-one really used it. :p So wouldn't it be possible to change the type to string? You can use an empty string to disable the ellipsis.

**Comment by patrick96** [comment]
It's technically possible to change the type, but it would break all existing configs that use it. Right now we only know that nobody who has it in their config noticed that it doesn't work. But `label-song-ellipsis = true` is part of the config shipped with polybar, so people would suddenly have 'true' added to their cut off labels.

What we will probably need to do is define another key, deprecate `label-*-ellipsis`, and maybe in the next major release use `label-*-ellipsis` the way you described

**Comment by kronn** [comment]
Isn't this issue resolved by merging #1198?
